.npmrc

```diff
@@ -1 +0,0 @@
-package-lock=false
```

Comment:
Don't do unrelated changes.

Reply:
this is required to run `npm audit` and npm recommends committing it